includes/abstracts/abstract-wc-stripe-payment-gateway.php

```diff
@@ -1486,6 +1513,16 @@ public function save_intent_to_order( $order, $intent ) {
 			$order->update_meta_data( '_stripe_setup_intent', $intent->id );
 		}
 
+		// Add the mandate id necessary for renewal payments with Indian cards.
+		if ( isset( $intent->charges ) && isset( $intent->charges->data ) ) {
```

Comment:
Here we can also use just one `isset`:

```suggestion
		if ( isset( $intent->charges->data ) ) {
```

Reply:
Applied in dbad167a07b43a1ad3203b0fccea7322272407c7